core/config.py

```diff
@@ -44,6 +45,11 @@ def __setitem__(self, key, value):
 __project_id = google.auth.default()[1]
 __version = os.getenv("GAE_VERSION")
 
+# Determine our basePath (as os.getCWD is broken on appengine)
+__project_base_path = str(Path().absolute())
+
+__core_base_path = globals()["__file__"].replace("/viur/core/config.py", "")
```

Comment:
Direct use of `__file__` should work as well (please verify!)
```suggestion
__core_base_path = __file__.replace("/viur/core/config.py", "")
```

Reply:
yes `__file__` works too.